gms/api/src/main/snapshot/com.linkedin.datajob.dataJobs.snapshot.json

```diff
@@ -438,9 +438,10 @@
             "name" : "AzkabanJobType",
             "namespace" : "com.linkedin.datajob.azkaban",
             "doc" : "The various types of support azkaban jobs",
-            "symbols" : [ "COMMAND", "HADOOP_JAVA", "HADOOP_SHELL", "HIVE", "PIG", "SQL" ],
```

Comment:
Let's not keep piling onto the `AzkabanJobType` 

cc @jjoyce0510 is the plan still to add a free form string?

Reply:
Yeah. This can either be a freeform string or another enum with a better name. 

I don't have a strong preference for adding a freeform string vs a better enum